content/en-us/reference/engine/classes/TextChatService.yaml

```diff
@@ -79,54 +79,75 @@ properties:
         <tbody>
+          <tr>
+            <td><b>RBXClearCommand</b></td>
+            <td>clear</td>
+            <td>cls</td>
+            <td>Clears the chat log for the local user.</td>
+            <td><code>/cls</code></td>
+          </tr>
           <tr>
             <td><b>RBXConsoleCommand</b></td>
             <td>console</td>
             <td></td>
             <td>Opens the Developer Console.</td>
-            <td><code>\console</code></td>
+            <td><code>/console</code></td>
           </tr>
           <tr>
             <td><b>RBXEmoteCommand</b></td>
             <td>emote</td>
             <td>e</td>
             <td>Plays an avatar emote.</td>
-            <td><code>\e dance</code></td>
+            <td><code>/e dance</code></td>
           </tr>
           <tr>
             <td><b>RBXHelpCommand</b></td>
             <td>help</td>
             <td>?</td>
             <td>Shows a list of chat commands.</td>
-            <td><code>\help</code></td>
+            <td><code>/help</code></td>
           </tr>
           <tr>
             <td><b>RBXMuteCommand</b></td>
             <td>mute</td>
             <td>m</td>
             <td>Mutes a user by their <code>Class.Player.Name|Name</code> or <code>Class.Player.DisplayName|DisplayName</code> in all <code>Class.TextChannel|TextChannels</code>.</td>
-            <td><code>\m Username</code></td>
+            <td><code>/m Username</code></td>
+          </tr>
+          <tr>
+            <td><b>RBXTeamCommand</b></td>
+            <td>team</td>
+            <td>t</td>
+            <td>Enters team chat mode where messages are only visible to teammates.</td>
+            <td><code>/t</code></td>
           </tr>
           <tr>
             <td><b>RBXUnmuteCommand</b></td>
             <td>unmute</td>
             <td>um</td>
             <td>Unmutes a user by their <code>Class.Player.Name|Name</code> or <code>Class.Player.DisplayName|DisplayName</code> in all <code>Class.TextChannel|TextChannels</code>.</td>
-            <td><code>\um Username</code></td>
+            <td><code>/um Username</code></td>
           </tr>
           <tr>
             <td><b>RBXVersionCommand</b></td>
             <td>version</td>
             <td>v</td>
             <td>Shows the chat version.</td>
-            <td><code>\version</code></td>
+            <td><code>/version</code></td>
+          </tr>
+          <tr>
+            <td><b>RBXWhisperCommand</b></td>
+            <td>whisper</td>
+            <td>w</td>
+            <td>Enters whisper mode with another <code>Class.Player</code>.</td>
+            <td><code>/w DisplayName</code> or <code>/w @Username</code></td>
           </tr>
         </tbody>
       </table>
 
       Note that you can edit, create, and remove
       `Class.TextChatCommand|TextChatCommands` even if
-      `Class.TextChatService:CreateDefaultCommands()|CreateDefaultCommands()` is
+      `Class.TextChatService.CreateDefaultCommands|CreateDefaultCommands` is
```

Comment:
Good catch. I probably quickly assumed that CreateDefaultCommands was a method (not a property) in a previous commit, so your fix is correct here.